create_graphics.py

```diff
@@ -447,10 +448,13 @@ def graphics_driver(cla):
     '''
 
     # Create an empty zip file
-    zipf = None
-    if cla.zip_dir:
-        os.makedirs(cla.zip_dir, exist_ok=True)
-        zipf = os.path.join(cla.zip_dir, 'files.zip')
+    zipf = []
+    for tile in cla.tiles:
+        if cla.zip_dir:
+            tile_zip_dir = os.path.join(cla.zip_dir, tile)
+            os.makedirs(tile_zip_dir, exist_ok=True)
+            tile_zip_file = os.path.join(tile_zip_dir, 'files.zip')
+            zipf.append(tile_zip_file)				
```

Comment:
The use of a dict for `zipf` would be more effective so that we'd know which tile each was associated with. Also naming it something plural is a bit more clear. Example:

```
zipfiles = {}
for tile in cla.tiles:
    ...
    zipfiles[tile] = tile_zip_file
```